src/reducers/appsMetricsReducer.js

```diff
@@ -31,6 +31,15 @@ export default function appsMetricsReducer(
 				error: action.error,
 			};
 		}
+		case APPS.REMOVE_APP_METRICS: {
```

Comment:
Why have you created multiple `REMOVE` actions? Ideally, we should create one `REMOVE` action on `APPS` and all the necessary reducers should listen to it, and act accordingly.